### ISSUE TITLE
custom `options.targets.day` is ignored and days which don't have the `day` class loose their `inactive` class

### ISSUE THREAD

**Issue by mbehzad** [kylestetz/CLNDR#330]
#### Context

When `options.constraints` is provided, during the updating of the `inactive` classes on the calendars controls, the day elements are targeted when the `options.targets.day` is anything different than the default `day` value. This leads to the day elements loosing the `inactive` css class.

the user should be allowed to use its css class names when using the `targets` options.
and in the script, the key name is compared with the options value and not the key as well.
#### Expected Behavior

Day elements should have the `inactive` css class when they are out of the `constraints`. Even if they have a css class other than `day` when it was announced via the `options.targets.day`.
#### Actual Behavior

Day elements loose the `inactive` css class
#### Possible Fix

compare `target` value with `'day'` instead of `this.options.targets.day`
(a pull request will be provided)
#### Steps to Reproduce

1.
```
$('.parent-element').clndr({
  targets: {
      day: "my-day"
  }
  constraints: {
      startDate: "2018-06-10"
  }
})
```
2.
3.

#### Your Environment

* Version used:
1.4.7
* Browser Name and version:
Any (e.g. chrome 67)
* Operating System and version (desktop or mobile):
windows 10
* Link to your project:

**Comment by mikegioia** [comment]
PR in #331, needs review

**Comment by mikegioia** [comment]
This is now resolved via #340 , I'm going to close the PR too
